### ISSUE TITLE
About the ver

### ISSUE THREAD

**Issue by zxp19821005** [EdgeTX/buddy#62]
Will you please add the correctlly version of the package?
It's always 'latest' now.
I've ported the package to AUR [edgetx-buddy-bin](https://aur.archlinux.org/packages/edgetx-buddy-bin) , but the version was always '0.1.0'.

**Comment by Pancronos** [comment]
Agree. I think that requires some changes, for example in translation is a fixed word "(Latest)": " but of course is very helpful to have a version number, users think to have the latest version and it well could be that is not.

**Comment by pfeerick** [comment]
That would require a tagged/versioned release, however, there hasn't been one, hence why it is always latest.

**Comment by raphaelcoeffic** [comment]
@pfeerick should we start incrementing the release numbers and have some proper "releases"? It would probably make sense for downstream packaging.

**Comment by pfeerick** [comment]
No complaints from the peanut gallery... especially if it makes the lives of potential contributors or other package maintainers better 🤭

**Comment by freshollie** [comment]
I'm happy to implement automated versioning. I've done this before using [semantic-release](https://github.com/marketplace/actions/action-for-semantic-release) which will auto increment the version number on merge to main depending on the conventional commitments which were merged to main

**Comment by raphaelcoeffic** [comment]
> I'm happy to implement automated versioning.

Sounds great :-) Let do it!

**Comment by HThuren** [comment]
Where did this end ?

**Comment by raphaelcoeffic** [comment]
> Where did this end ?

Nowhere for now 😅
